### PR TITLE
WebGL:  TypsScript for WebGLBackground.js and WebGLUtils.js

### PR DESCRIPTION
**Pull request by linbingquan**

### CODE REVIEW

**Comment by mrdoob** [comment]
Thanks!

File: src/renderers/webgl/WebGLUtils.d.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export class WebGLUtils {
+	constructor(gl: CanvasRenderingContext2D | WebGLRenderingContext, extensions: any, capabilities: any);
```
Comment (Mugen87) [inline src/renderers/webgl/WebGLUtils.d.ts]: I think it's wrong to use `CanvasRenderingContext2D` which represent a 2D drawing context. You always create a WebGL or WebGL2 context. Hence it should be 

`gl: WebGLRenderingContext | WebGL2RenderingContext`

Comment (Mugen87) [reply src/renderers/webgl/WebGLUtils.d.ts]: BTW: This is also wrong in `WebGLAnimation.d.ts` and `WebGLAttributes.d.ts`.

Comment (Mugen87) [reply src/renderers/webgl/WebGLUtils.d.ts]: Solved via #16458 😊

**Pull request merged**